example/constants/styles.ts

```diff
@@ -0,0 +1,105 @@
+import { StyleSheet } from 'react-native'
+import { Colors } from './colors'
+
+export const GlobalStyles = StyleSheet.create({
+  screenContainer: {
+    flex: 1,
+    backgroundColor: Colors.swmBlue,
+  },
+  centeredContainer: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 20,
+    backgroundColor: '#f5f5f5',
```

Comment:
I’d prefer to avoid using colors other than those defined in the Colors constants.

```suggestion
    backgroundColor: Colors.white,
```